src/ucp/core/ucp_ep.c

```diff
@@ -1411,7 +1404,7 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
 
     status = ucp_ep_config_key_copy(&config->key, key);
     if (status != UCS_OK) {
-        return status;
+        goto err;
```

Comment:
use label for single reference?
what is wrong with return status?

Reply:
yes, so all error flow would go thru same place